### ISSUE TITLE
Fix Unsupported engine

### ISSUE THREAD

**Issue by mauriciabad** [strapi-community/plugin-todo#4]
Fix the following error when installing the package using node 20.

```log
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'strapi-plugin-todo@0.0.9',
npm WARN EBADENGINE   required: { node: '>=12.x.x <=16.x.x', npm: '>=6.0.0' },
npm WARN EBADENGINE   current: { node: 'v20.5.1', npm: '9.8.1' }
npm WARN EBADENGINE }
```

I opened a PR fixing it https://github.com/remidej/strapi-plugin-todo/pull/3

**Comment by MauriceArikoglu** [comment]
@remidej Same for node 18. Any reason the plugin only supports until node 16?

**Comment by remidej** [comment]
There was no particularly, it's just the boilerplate from when I created the plugin, sorry about that. It should now be fixed in version 0.0.10
